### PR TITLE
Improvements to resources tutorial

### PR DESCRIPTION
**Pull request by mattj-io**
Fix some bugs, refactor, improve content. Addresses DOCS-2059

## Description
https://jira.mesosphere.com/browse/DOCS-2059

## Urgency
- [ ] Blocker 
- [ ] High
- [ x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.10/tutorials/dcos-101/resources.md [hunk 1]
```diff
@@ -34,67 +35,160 @@ Resource management and resource isolation between tasks are core challenges for
...
+### Scale horizontally by increasing the instance count
+
+Horizontal scaling involves increasing the number of instances of an application. You can scale the instance count in two ways:
+
+1. Scale an entire app group by a factor. 
+1. Directly set number of instances for an app.
+
+**Scale dcos-101 application group:** 
+
+Since both appl and app2 share the same app group, we can scale them together
```
Comment (sascala) [inline 1.10/tutorials/dcos-101/resources.md]: period at the end of the sentence.

File: 1.10/tutorials/dcos-101/resources.md [hunk 2]
```diff
@@ -34,67 +35,160 @@ Resource management and resource isolation between tasks are core challenges for
...
 
+The problem here is that there are no resource offers large enough to match the request.
+
+Solution: 
+
+* Provision larger nodes or scale the app down to a level where it fits onto the free resources on a single node:
+
+`dcos marathon app update /dcos-101/app2 --force cpus=1`
+
+Note that you have to use the force flag again.
```
Comment (sascala) [inline 1.10/tutorials/dcos-101/resources.md]: **Note:** You must use the force flag.

File: 1.10/tutorials/dcos-101/resources.md [hunk 3]
```diff
@@ -34,67 +35,160 @@ Resource management and resource isolation between tasks are core challenges for
...
+
+# Debugging resource isolation
+
+What happens if an app tries to use more resources then it is allocated? The most common problem is memory consumption in conjunction with JVM-based applications.
+
+To simulate this:
+
+* Deploy the [memory eater](https://github.com/joerg84/dcos-101/blob/master/oomApp/oomApp.go) app
```
Comment (sascala) [inline 1.10/tutorials/dcos-101/resources.md]: period at the end.

**Comment by mattj-io** [comment]
@sascala suggestions changed

**Pull request merged**